sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/controller/ApplicationController.java

```diff
@@ -95,15 +98,15 @@
     applicationFilter.setName(name);
     applicationFilter.setOwner(owner);
 
-    PageableResult pageableResult = new PageableResult(size, offset);
+    PageableResult pageableResult = new PageableResult(size, offset, searchCookie);
 
     PageResult<Application> foundApplications =
         applicationService.findByProperties(realm, applicationFilter, pageableResult);
 
     if (foundApplications.isHasMoreResult()) {
       URI location =
           ServletUriComponentsBuilder.fromCurrentRequest()
-              .replaceQueryParam("offset", offset + size)
+              .replaceQueryParam("searchToken", foundApplications.getSearchToken())
```

Comment:
offset seams no more used in this implementation. maybe take it of from the request param ? and nextstart as well ? we would rely only on the token
if nextStart or offset is still useful the offset should still be changed in this url

Reply:
You can still make a request with an offset and then use the searchtoken to continue the search

Reply:
ok

Reply:
offset is not working anymore

Reply:
yep, it's normal, offset is only available with vlv request whereas cookie is only avalaible for "classic request"